addons/mail/static/src/env/env.js

```diff
@@ -76,7 +84,7 @@ env.bus.on(
     () => env.messagingBus.trigger('will_show_home_menu')
 );
 
-(async function start() {
+async function createMessaging() {
```

Comment:
Is it possible to defined this method before the object.assign on env, so that `messagingCreatedPromise` can be directly set to its correct value?

Reply:
yes